ui/app/src/components/DateTimePicker/DateTimePicker.tsx

```diff
@@ -109,9 +115,12 @@ function DateTimePicker(props: DateTimePickerProps) {
   // displayed when the time picker is opened. `internalDate` is a transposed date/time
   // used to sync the timepicker with what's displayed to the user.
   const internalDate = useMemo(() => {
-    const date = value ? new Date(value) : new Date();
-    const localOffset = moment().format().substr(-6);
-    const dateWithoutOffset = moment(date).tz(timeZone).format().substr(0, 19);
+    if (nou(value)) {
+      return null;
+    }
+    const date = value ? new Date(value) : null;
```

Comment:
This line seems like a redundant check based on the previous lines conditional statement